guides/plugins/plugins/administration/modify-blacklist-for-dynamic-product-groups.md

```diff
@@ -0,0 +1,89 @@
+## Overview
+
+The module "Dynamic product groups" includes a condition builder to properly configure your dynamic product groups.
+You might have noticed though, that this condition builder does not show all available properties,
+since some of them are blacklisted in the code, such as e.g. `createdAt`.
+
+In this Guide you'll get two quick examples on how to either add new properties to this blacklist or even remove
+properties from the blacklist, so they're actually shown in the administration and thus can be used.
+
+## Prerequisites
+
+This Guide **will not** explain in detail how to override an existing component.
+For this Guide you'll have to extend the component [sw-product-stream-field-select](https://github.com/shopware/platform/blob/trunk/src/Administration/Resources/app/administration/src/module/sw-product-stream/component/sw-product-stream-field-select/index.js) though, since it's the one [actually checking for the properties in the computed property options](https://github.com/shopware/platform/blob/trunk/src/Administration/Resources/app/administration/src/module/sw-product-stream/component/sw-product-stream-field-select/index.js#L41).
```

Comment:
```suggestion
For this guide you'll have to extend the component [sw-product-stream-field-select](https://github.com/shopware/platform/blob/v6.3.4.1/src/Administration/Resources/app/administration/src/module/sw-product-stream/component/sw-product-stream-field-select/index.js) though, since it's the one [actually checking for the properties in the computed property options](https://github.com/shopware/platform/blob/v6.3.4.1/src/Administration/Resources/app/administration/src/module/sw-product-stream/component/sw-product-stream-field-select/index.js#L41).
```